README.md

```diff
@@ -108,9 +242,37 @@ These are built in CI. See the workflow files for the latest `earthly` commands:
 - [node](.github/workflows/main.yml)
 - [toolkit](.github/workflows/main.yml)
 
-### Start bootstrapped local network
+### Start local network
+
+**Available Networks:**
+- `local` - Development network (default)
+- `qanet` - QA testing network
+- `preview` - Preview/staging network
+- `perfnet` - Performance testing network
+
+Chain specifications are located in `/res/` directory.
+
+**Configuration Parameters:**
+
+| Parameter | Environment Variable | CLI Flag | Description |
+|-----------|---------------------|----------|-------------|
+| Config preset | `CFG_PRESET=dev` | - | Development mode configuration |
+| Validator seed | `SEED=//Alice` | - | Validator identity (//Alice, //Bob, etc.) |
+| Chain spec | - | `--chain local` | Network to connect to |
+| Base path | - | `--base-path /tmp/node-1` | Data directory |
+| Validator mode | - | `--validator` | Run as validator |
+| P2P port | - | `--port 30333` | Networking port (default: 30333) |
+| RPC port | - | `--rpc-port 9944` | WebSocket RPC port (default: 9944) |
+| Node key | - | `--node-key "0x..."` | Network identity key |
+| Bootstrap nodes | - | `--bootnodes "/ip4/..."` | Initial peers |
```

Comment:
Better to use the `BOOTNODES` env-var (space-separated list): https://github.com/midnightntwrk/midnight-node/blob/5c1293bfe0f4940fdff62d10165aeb9775fa9423/node/src/cfg/substrate_cfg/mod.rs#L43-L44